services/storage/objects/serverless.yml

```diff
@@ -15,14 +15,6 @@ provider:
 
```

Comment:
This file should not have been changed ;)

Reply:
Pretty bad, sorry